packages/evm/.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "printWidth": 100,
+  "singleQuote": true,
+  "trailingComma": "all",
+  "useTabs": false,
+  "arrowParens": "avoid"
+}
```

Comment:
I think some files are not formatted correctly, but let's do the prettier after we merge the PR as we will have a lot of conflicts due to reformatting

Reply:
@wainola also informed me about these